packages/terra-time-input/src/orion-fusion-theme/TimeInput.module.scss

```diff
@@ -39,6 +39,8 @@
     --terra-time-input-focus-ltr-box-shadow: 0 0 8px #a6d9f4;
     --terra-time-input-focus-rtl-box-shadow: 0 0 8px #a6d9f4;
     --terra-time-input-focus-color: #1c1f21;
+    --terra-time-input-outline: 2px dashed #000;
```

Comment:
We are not making dashed outline changes for fusion theme for other components can you refer to other components that has got outline styles

Reply:
I have verified with other components (terra-tabs), there we are making dashed outline changes for fusion theme as well (https://github.com/cerner/terra-framework/blob/main/packages/terra-tabs/src/orion-fusion-theme/Tab.module.scss#L5)

Reply:
but we have not made similar changes for date-input, terra-button etc and we can also see the same on some of the newly added terra-framework components like terra-compact-interactive-list, terra-flowsheet-data-grid
Unless it is specified by UX team I think we should not update the fusion styles

Reply:
Updated - https://github.com/cerner/terra-framework/pull/1966/commits/78f8257e199c07a68c38a6706123a196bdeb5863#diff-3610f3572ad007f8e3762cfe754e680935cd824c5a140ae9ef76e03b82e55259